src/ethereum-json-rpc-client/src/http_outcall.rs

```diff
@@ -71,14 +139,16 @@ impl Client for HttpOutcallClient {
                     value: "application/json".to_string(),
                 },
             ];
+            log::trace!("Making http request to {url} with headers: {headers:?}");
+            log::trace!("Request body is: {}", String::from_utf8_lossy(&body));
 
             let request = CanisterHttpRequestArgument {
                 url,
                 max_response_bytes,
                 method: HttpMethod::POST,
                 headers,
                 body: Some(body),
-                transform: Some(TransformContext::from_name("transform".to_string(), vec![])),
+                transform: None,
```

Comment:
Should this be `transform: self.transform_context`?

Reply:
You are absolutely right